### PR TITLE
Implemented Document upload screens and necessary functions for web.

### PR DESCRIPTION
**Pull request by Vedantfuley**
Document upload screen for the Web is implemented where you can select the images from the local storage and upload.

### CODE REVIEW

**Comment by Anik-Chakraborty** [comment]
image
upload image both gallery or camera doesn't work

**Comment by Vedantfuley** [comment]
Actually on my side its working.

https://github.com/liveasy97/transporterAppFlutter/assets/110230391/8e50bfb7-1ef5-4ca3-87fb-6b511649e4fe

**Comment by Anik-Chakraborty** [comment]
getting this error while clicking or gallery or camera
image

**Review by Anik-Chakraborty — APPROVED** [review]

File: lib/screens/TransporterOrders/docInputPod.dart [hunk 1]
```diff
@@ -327,10 +467,10 @@ class _docInputPodState extends State<docInputPod> {
                       ElevatedButton(
                           style: ButtonStyle(
                               backgroundColor: MaterialStateProperty.all<Color>(
-                                  Color(0xff0077B6))),
+                                  const Color(0xff0077B6))),
```
[the remark this answers is not in the document]
Comment (Vedantfuley) [reply lib/screens/TransporterOrders/docInputPod.dart]: Done.

**Pull request merged**